modules/logging-release-notes-5-9-7.adoc

```diff
@@ -0,0 +1,30 @@
+// module included in logging-5-9-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="cluster-logging-release-notes-5-9-7_{context}"]
+= Logging 5.9.7
+This release includes link:https://access.redhat.com/errata/RHBA-2024:7324[OpenShift Logging Bug Fix Release 5.9.7].
+
+
+[id="openshift-logging-5-9-7-bug-fixes_{context}"]
+== Bug fixes
+
+* Before this update, the `clusterlogforwarder.spec.outputs.http.timeout` parameter was not applied to the Fluentd configuration when Fluentd was used as the collector type, causing HTTP timeouts to be misconfigured. With this update, the `clusterlogforwarder.spec.outputs.http.timeout` parameter is now correctly applied, ensuring Fluentd honors the specified timeout and handles HTTP connections according to the user’s configuration. (link:https://issues.redhat.com/browse/LOG-6125[LOG-6125])
+
+* Before this update, the TLS section was added without verifying the broker URL schema, resulting in SSL connection errors if the URLs did not start with `tls`. With this update, the TLS section is now added only if the broker URLs start with `tls`, preventing SSL connection errors. (link:https://issues.redhat.com/browse/LOG-6041[LOG-6041])
+
+[id="openshift-logging-5-9-7-CVEs_{context}"]
+== CVEs
+
+* link:https://access.redhat.com/security/cve/CVE-2024-6104[CVE-2024-6104]
+* link:https://access.redhat.com/security/cve/CVE-2024-6119[CVE-2024-6119]
+* link:https://access.redhat.com/security/cve/CVE-2024-34397[CVE-2024-34397]
+* link:https://access.redhat.com/security/cve/CVE-2024-45296[CVE-2024-45296]
+* link:https://access.redhat.com/security/cve/CVE-2024-45490[CVE-2024-45490]
+* link:https://access.redhat.com/security/cve/CVE-2024-45491[CVE-2024-45491]
+* link:https://access.redhat.com/security/cve/CVE-2024-45492[CVE-2024-45492]
+* link:https://access.redhat.com/security/cve/CVE-2024-45801[CVE-2024-45801]
+
+[NOTE]
+====
+For detailed information on Red Hat security ratings, review link:https://access.redhat.com/security/updates/classification/#moderate[Severity ratings].
```

Comment:
It's normally "information about" not "information on". https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i look for "information about".

However if previous RNs have "information on", I'd suggest keeping this minor style violation for the sake of consistency.